implementations/rust/ockam/ockam_command/src/vault/mod.rs

```diff
@@ -41,6 +41,7 @@ pub enum VaultSubcommand {
         /// Name of the vault
         name: Option<String>,
     },
+    List {},
```

Comment:
Would be nice to add a simple test to `commands.bats`, as part of the `"create a vault and do show on it"` test.

Reply:
Added the according test in `commands.bat`.